crawl-ref/source/ng-wanderer.cc

```diff
@@ -6,6 +6,7 @@
 #include "item-use.h"
 #include "items.h"
 #include "jobs.h"
+#include "makeitem.h"
```

Comment:
Is there a reason this include is here? The code seems to build properly with it removed.

Reply:
Huh, I thought it was necessary to include the definition of `handle_generated_misc`. Is it not always necessary to include header files for all functions called within the file? I can certainly remove the include.

Reply:
Oh nevermind I just added it to the wrong file, oops.